### ISSUE TITLE
Speed up CI/CD

### ISSUE THREAD

**Issue by StachuDotNet** [darklang/dark#3833]
- [x] remove ocaml production containers
- [x] remove ocaml from devcontainer
- [x] remove stroller, build-stroller job, and tests
- [x] remove backend build-step
- [x] move integration tests to a single big machine
- [ ] optimize build-fsharp-backend if possible
  - [ ] do we need to compress blazor if we're going to compress it when uploading?
  - [x] use a bigger machine?
- [x] parallelize gcp-containers-test
- [x] parallelize push-to-gcp

**Comment by pbiggar** [comment]
- [ ] I think we don't need to attach the workspace during deploy

**Comment by pbiggar** [comment]
I've done as much as I can right now, so waiting until we're done removing ocaml entirely before we can move forward.

**Comment by pbiggar** [comment]
Blazor is gone, new work on this should be in a new issue.
